plugins/permission-common/report.api.md

```diff
@@ -183,9 +183,19 @@ export class PermissionClient implements PermissionEvaluator {
   ): Promise<QueryPermissionResponse[]>;
 }
 
+// @public
+export type PermissionClientOptions = {
+  discovery: DiscoveryApi;
+  config: Config;
+  loaderCacheTtl?: number;
+  cacheTtl?: number;
+  batchDelay?: number;
```

Comment:
A bit unfortunate to have these here. They're for tests right? Can you make it a dedicated `@internal` static `forTests` method or something like that instead?

Reply:
Yeah, for tests. I will refactor these out of the public API.